src/parameters/parameters_flow_solver.cpp

```diff
@@ -86,25 +88,59 @@ void FlowSolverParam::declare_parameters(dealii::ParameterHandler &prm)
 
         prm.enter_subsection("grid");
         {
-          prm.declare_entry("input_mesh_filename", "naca0012",
-                            dealii::Patterns::FileName(dealii::Patterns::FileName::FileType::input),
-                            "Filename of the input mesh: input_mesh_filename.msh. For cases that import a mesh file.");
+            prm.declare_entry("input_mesh_filename", "naca0012",
+                              dealii::Patterns::FileName(dealii::Patterns::FileName::FileType::input),
+                              "Filename of the input mesh: input_mesh_filename.msh. For cases that import a mesh file.");
 
-          prm.declare_entry("grid_degree", "1",
-                            dealii::Patterns::Integer(1, dealii::Patterns::Integer::max_int_value),
-                            "Polynomial degree of the grid. Curvilinear grid if set greater than 1; default is 1.");
+            prm.declare_entry("grid_degree", "1",
+                              dealii::Patterns::Integer(1, dealii::Patterns::Integer::max_int_value),
+                              "Polynomial degree of the grid. Curvilinear grid if set greater than 1; default is 1.");
+
+            prm.declare_entry("grid_left_bound", "0.0",
+                              dealii::Patterns::Double(0, dealii::Patterns::Double::max_double_value),
+                              "Left bound of domain for hyper_cube mesh based cases.");
+
+            prm.declare_entry("grid_right_bound", "1.0",
+                              dealii::Patterns::Double(0, dealii::Patterns::Double::max_double_value),
+                              "Right bound of domain for hyper_cube mesh based cases.");
+
+            prm.declare_entry("number_of_grid_elements_per_dimension", "4",
+                              dealii::Patterns::Integer(1, dealii::Patterns::Integer::max_int_value),
+                              "Number of grid elements per dimension for hyper_cube mesh based cases.");
 
-          prm.declare_entry("grid_left_bound", "0.0",
-                            dealii::Patterns::Double(0, dealii::Patterns::Double::max_double_value),
-                            "Left bound of domain for hyper_cube mesh based cases.");
+            prm.declare_entry("num_refinements", "0",
+                              dealii::Patterns::Integer(0, dealii::Patterns::Integer::max_int_value),
+                              "Number of refinements to be performed.");
```

Comment:
A more detailed description? Maybe "Number of mesh refinements for Gaussian Bump and NACA0012 mesh based cases."

Reply:
fixed